packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -941,6 +943,16 @@ mixin WidgetInspectorService {
     _errorsSinceReload = 0;
   }
 
+  bool setStructuredErrorsEarly() {
+    return const bool.fromEnvironment('structuredErrors');
+  }
+
+  void setStructuredErrors() {
```

Comment:
this shouldn't be a setter either. A setter should typically take a parameter with a new value. 
This method is more along the lines of
`initStructuredErrorHandler` or similar.